src/nfa/operations.cc

```diff
@@ -1208,6 +1202,12 @@ std::set<mata::Word> mata::nfa::Nfa::get_words(unsigned max_length) {
     return result;
 }
 
+OrdVector<Symbol> mata::nfa::get_symbols_to_work_with(const Nfa& nfa, const mata::Alphabet *const shared_alphabet) {
```

Comment:
Shouldn't this return `const OrdVector<Symbol>&`?

Reply:
No. It is constructing the set of symbols, either from alphabets (which themselves do not *necessarily* have a handy `OrdVector<Symbol>` allocated either), or constructing a new set directly from the transition relation. Therefore, a new set has to be allocated.